mapr/manifest.json

```diff
@@ -0,0 +1,29 @@
+{
+  "display_name": "MapR",
+  "maintainer": "help@datadoghq.com",
+  "manifest_version": "1.0.0",
+  "name": "mapr",
+  "metric_prefix": "mapr.",
+  "metric_to_check": "mapr.metrics.submitted",
+  "creates_events": false,
+  "short_description": "Collect the monitoring metrics made available by MapR.",
+  "guid": "7d1de422-85a6-47cc-9962-427a9499d109",
+  "support": "core",
+  "supported_os": [
+    "linux"
+  ],
+  "public_title": "Datadog-MapR Integration",
+  "categories": [
+    "data-store",
+    "os & system",
+    "processing"
```

Comment:
Should we keep only `data-store` ?

Reply:
This is simlar to what we have for hadoop related stuff, MapR being an ecosystem does all those things.

`data-store` because database and queues
`os & system` because filesystem
`processing` because spark, yarn, mapreduce.